streampark-console/streampark-console-webapp/src/views/system/variable/View.vue

```diff
@@ -0,0 +1,347 @@
+    the License.  You may obtain a copy of the License at
+
+       https://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+
+<template>
+  <a-card :bordered="false">
+    <div
+      class="table-page-search-wrapper">
+      <a-form
+        layout="inline">
+        <a-row
+          :gutter="48">
+          <div
+            class="fold">
+            <a-col
+              :md="8"
+              :sm="24">
+              <a-form-item
+                label="Variable Code"
+                :label-col="{span: 4}"
+                :wrapper-col="{span: 18, offset: 2}">
+                <a-input
+                  v-model="queryParams.variableCode" />
+              </a-form-item>
+            </a-col>
+            <a-col
+              :md="8"
+              :sm="24">
+              <a-form-item
+                label="Description"
+                :label-col="{span: 4}"
+                :wrapper-col="{span: 18, offset: 2}">
+                <a-input
+                  v-model="queryParams.description" />
+              </a-form-item>
+            </a-col>
+          </div>
+          <a-col
+            :md="8"
+            :sm="24">
+            <span
+              class="table-page-search-bar">
+              <a-button
+                type="primary"
+                shape="circle"
+                icon="search"
+                @click="search" />
+              <a-button
+                type="primary"
+                shape="circle"
+                icon="rest"
+                @click="reset" />
+              <a-button
+                type="primary"
+                shape="circle"
+                icon="plus"
+                v-permit="'variable:add'"
+                @click="handleAdd" />
+            </span>
+          </a-col>
+        </a-row>
+      </a-form>
+    </div>
+
+    <!-- 表格区域 -->
+    <a-table
+      ref="TableInfo"
+      :columns="columns"
+      :data-source="dataSource"
+      :pagination="pagination"
+      :loading="loading"
+      :scroll="{ x: 900 }"
+      @change="handleTableChange">
+      <template
+        slot="email"
+        slot-scope="text">
+        <a-popover
+          placement="topLeft">
+          <template
+            slot="content">
+            <div>
+              {{ text }}
+            </div>
+          </template>
+          <p
+            style="width: 150px;margin-bottom: 0">
+            {{ text }}
+          </p>
+        </a-popover>
+      </template>
+      <template
+        slot="operation"
+        slot-scope="text, record">
+        <svg-icon
+          v-permit="'variable:update'"
+          v-if="(record.username !== 'admin' || userName === 'admin')"
```

Comment:
This condition is wired. I guess you copy from user management, right?

Admin user cannot allow to change, but variable should can be update even if it is created by admin.

Reply:
Your review is very detailed, thank you very much, I will change it immediately